types/silenced.proto

```diff
@@ -22,7 +22,7 @@ message Silenced {
 
   // ExpireOnResolve defaults to false, clears the entry on resolution when set
   // to true
-  bool expire_on_resolve = 3 [(gogoproto.nullable) = true];
+  bool expire_on_resolve = 3 [(gogoproto.nullable) = true, (gogoproto.jsontag) = "expire_on_resolve"];
```

Comment:
This one can stay as it is. It's a pointer, so it's zero-value is nil. If the value pointed to is false, it will be displayed.

Reply:
Yeah, looks like even with `(gogoproto.nullable) = true` the bool is not a pointer. I forgot about that. How annoying that pointers can't be forced with proto3.

Reply:
Agreed..